pandas/core/indexes/multi.py

```diff
@@ -1035,11 +1035,11 @@ def is_lexsorted(self):
         """
         return self.lexsort_depth == self.nlevels
 
-    def is_lexsorted_for_tuple(self, tup):
+    def _true_slices_in_tup(self, tup):
         """
-        Return True if we are correctly lexsorted given the passed tuple
+        Return indices of (non-trivial) slices in "tup"
```

Comment:
this should now say a boolean list, same length as the input tuple.

move to ``pandas.core.common``, after ``is_null_slice``. make a similarly named, maybe ``is_true_slices``